R/pin-read-write.R

```diff
@@ -64,6 +64,10 @@ pin_read <- function(board, name, version = NULL, hash = NULL, ...) {
 #'   use the default for `board`
 #' @param tags A character vector of tags for the pin; most important for
 #'   discoverability on shared boards.
+#' @param check_hash Check whether the pin contents are identical to the last
```

Comment:
Should we consider defaulting this to `TRUE`?

Reply:
Yeah, let's try it. A release is still a little ways out so we can see if it is a problem in an expected way.

Reply:
Oh actually @hadley the other reason I had not set it to `TRUE` originally is it adds one additional call to `pin_meta()`. Do we think this is OK? All the API calls can be especially painful on Connect, but maybe adding yet one more set isn't much marginal difference.

Reply:
Hmmmm, I think it's probably still worth it. It definitely saves you a lot of time/space in the case where the data hasn't changed. We'll definitely need to promote that change heavily.